frontend/src/components/permissions/helpers/get-permission-by-name/get-permission-by-name.ts

```diff
@@ -0,0 +1,10 @@
+import { IPermission } from 'common/interfaces';
```

Comment:
Rename to `frontend/src/components/permissions/helpers/get-permission-by-name/get-permission-by-name.helper.ts`